_i18n/de/resources/moneropedia/paperwallet.md

```diff
@@ -1,9 +1,9 @@
 ---
-terms: ["paperwallet", "paperwallets", "paper-wallet", "paper-wallets"]
-summary: "A paper wallet stores the information necessary to send and receive Monero"
+terms: ["paperwallet", "paperwallets", "paper-wallet", "paper-wallets", "Paper-Wallet", "Paper-Wallets"]
+summary: "Ein Paper-Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A paper wallet stores the information necessary to send and receive Monero.
+Ein Paper-Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen.
```

Comment:
Translation is fine, but the entry on paper wallets is way too short to be useful.